src/tools/storage-perf/StoragePerfTool.cpp

```diff
@@ -51,6 +82,7 @@ class Perf {
         CHECK(mClient_->waitForMetadReady());
         client_ = std::make_unique<StorageClient>(threadPool_, mClient_.get());
         time::Duration duration;
+        static uint32_t interval = 1;
```

Comment:
I guess you mean 100ms here? According to my understanding, you split 1 second to 10 * 100ms, and set corresponding qps of each slot.

Reply:
Ah, I got it.

Reply:
10 ms is a cycle. `slots_` is the sending plan for a cycle. So it should sleep 1 ms.